packages/workspace/src/type.ts

```diff
@@ -79,13 +79,20 @@ export interface WorkspaceRegistry {
   [WorkspaceFlavour.LOCAL]: LocalWorkspace;
 }
 
+interface CRUDOptions {
+  local: boolean;
```

Comment:
I think this should be removed

Reply:
I think upstream (which is next.js layer) shouldn't care about the detail of the loading logic in each plugin